pkg/nodeconfig/nodeconfig.go

```diff
@@ -130,6 +131,48 @@ func discoverKubeAPIServerEndpoint() (string, error) {
 	return host.Status.APIServerInternalURL, nil
 }
 
+// getWICDCredentials returns the CA cert and access token associated with the WICD service account
```

Comment:
I suggest moving this function into `init.go` as that is the only place it is used